fossunited/www/payment-success.html

```diff
@@ -0,0 +1,129 @@
+{% extends "templates/foss_base.html" %} {% block title %} {{ _("Your Payment
+Was Successful!") }}{% endblock %} {% block page_content %}
+<div id="payment-success-page">
+  <div class="payment-success">
+    <div id="success-icon-div">
+      <svg class="success-icon container-center icon">
+        <use href="#icon-solid-success"></use>
+      </svg>
+    </div>
+    <div class="text-center">
+      <p>
+        Your Payment was successful!!
+        <br />
+        You will receive a copy of your ticket on your email.
```

Comment:
D   Did we set up a print format for this as well? Or is it in process?

Reply:
It is process. I will have to do that on the desk

Reply:
@harshtandiya isn't it good to have it as a standard?